src/main/java/frc/robot/RobotContainer.java

```diff
@@ -20,12 +26,16 @@
  */
 public class RobotContainer {
   // The robot's subsystems and commands are defined here...
+    public static final XboxController xboxController = new XboxController(1);
+    private final Trigger RT = new Trigger(() -> xboxController.getRawAxis(XboxController.Axis.kRightTrigger.value) > 0.3);
+    Shooter sniper = new Shooter();
```

Comment:
```suggestion
    private final XboxController xboxController = new XboxController(1);
    private final Trigger RT = new Trigger(() -> xboxController.getRawAxis(XboxController.Axis.kRightTrigger.value) > 0.3);
    private final Shooter sniper = new Shooter();
```

And rename `sniper` to `shooter`